javascript/src/live/helpers.js

```
@@ -87,21 +96,64 @@ export function initMicrophoneRecorder(config, onAudioChunk, onEnd) {
     const chunkSize = chunkDuration * bytesPerSecond;
-    const buffer = readFileSync(resolve(filePath))
-        // remove wav header
-        .subarray(44);
     const recorder = {
         interval: null,
         start() {
             let offset = 0;
             this.interval = setInterval(() => {
-                onAudioChunk(buffer.subarray(offset, (offset += chunkSize)));
-                if (offset >= buffer.length) {
+                onAudioChunk(audioData.subarray(offset, (offset += chunkSize)));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)